docs/usage/recreate.rst

```diff
@@ -13,7 +13,7 @@ Examples
     $ borg create /mnt/backup::archive /some/files --compression lz4
     # Then compress it - this might take longer, but the backup has already completed,
     # so no inconsistencies from a long-running backup job.
-    $ borg recreate /mnt/backup::archive --recompress --compression zlib,9
+    $ borg recreate /mnt/backup::archive --recompress if-different --compression zlib,9
```

Comment:
this does not change the behaviour as this is the same as the `const` value, see argparse docs.